tests/playwright/sanity/plugin-tester-generator.ts

```diff
@@ -82,8 +82,20 @@ export const generatePluginTests = ( testType: string ) => {
 				}
 				await page.goto( '/law-firm-about/?elementor' );
 
-				await editor.getPreviewFrame().getByRole( 'heading', { name: 'About Us' } ).waitFor( { timeout: 15000 } );
+				try {
+					await editor.getPreviewFrame().getByRole( 'heading', { name: 'About Us' } ).waitFor( { timeout: 10000 } );
+				} catch ( error ) {
+					await page.reload();
+					await editor.getPreviewFrame().getByRole( 'heading', { name: 'About Us' } ).waitFor( { timeout: 10000 } );
+				}
+
 				await wpAdmin.closeAnnouncementsIfVisible();
+
+				if ( 'the-plus-addons-for-elementor-page-builder' === plugin.pluginName ) {
```

Comment:
Should we generalize this and add it as a new parameter in pluginList?

Reply:
@yotamselementor This is a specific popup for this plugin only that's why I think it's not worth it